src/backoffice/shared/components/debug/debug.modal.element.ts

```diff
@@ -0,0 +1,163 @@
+import { css, html, nothing, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { UUITextStyles } from '@umbraco-ui/uui-css';
+import { UmbModalHandler, UmbModalLayoutElement } from '@umbraco-cms/modal';
+
+@customElement('umb-debug-modal-layout')
+export class UmbDebugModalLayout extends UmbModalLayoutElement {
```

Comment:
We need to send in a generic here to imply what the data might be

```suggestion
export class UmbDebugModalLayout extends UmbModalLayoutElement<any> {
```

would do for now